documents/operating.md

```diff
@@ -29,15 +29,112 @@ SECTION 2: 🤝🤝🤝 Partnership 🤝🤝🤝
 ---------------------------------------------------------------------
 SECTION 3: 🤔🤔🤔 Projects 🤔🤔🤔
 ---------------------------------------------------------------------
-- [ ] Types of project
-- [ ] Evaluating ideas
-- [ ] IP (Intellectual Property / Internet Protocol)
-- [ ] Open source
-- [ ] When a project should be considered retired
-- [ ] Removing a project from BIFFUD
+- [x] Types of project
+- [x] Evaluating ideas
+- [x] IP (Intellectual Property / Internet Protocol)
+- [x] Open source
+- [x] When a project should be considered retired
+- [x] Removing a project from BIFFUD
 - [ ] Adopting IP from existing entities / projects
 - [ ] Reallocating copyright / IP upon spinoff
 
+### BIFFUD Projects
+BIFFUD exists to support the creation of projects.  These projects are
+approved by the Corporate Overlords, owned and maintained by Members,
+and built by Minions.
+
+### Ownership Contracts
+All owners of a project must be BIFFUD members, and the percentages and specific
+terms of ownership must be coordinated by those members as part of the project's
+induction into The Company in the form of an Ownership Contract.
+
+The Ownership Contract should specify participation expectations
+and ownership percentages, as well as guidelines for how potential profits
+should be distributed.
+
+Changes to project ownership over time must be communicated to The Beastmaster.
+
+### Project Resources
+Resources required to complete or support a project can come from several
+different sources:
+
+A. **Sponsored support** includes resources from grants, gifts, or other payment
+from third parties.  Resources provided by a third party must be approved by the
+Overmind and must be distributed as per the third party's agreement.
+
+B. **Member Support** includes resources directly provided by BIFFUD members;
+this may or may not be associated with ownership.
+
+C. **BIFFUD Support** includes resources provided by the entire corporation.  These
+resources represent expense and affect profit calculations at the end of the year.
+
+D. **Profit Reinvestment** at the end of the fiscal year a project may be allocated
+funds based on profit earned.  These funds can be distributed among owners or re-invested
+into the project.
+
+
+### Disputes Between Project Owners
+If project owners disagree, the Beastmaster is responsible for addressing the
+disagreement.  If the beastmaster fails to create harmony, the Undermind, Overmind, and
+Bestmaster make the final determination.
```

Comment:
does this need clarification? Is there a formal debate + simple vote, trail by combat, Mario Party showdown??